ksql-common/src/main/java/io/confluent/ksql/function/UdfFactory.java

```diff
@@ -67,6 +67,11 @@ private void checkCompatible(final KsqlFunction ksqlFunction,
           + " as a function with the same name and argument types already exists "
           + functions.get(paramTypes));
     }
+    if (!ksqlFunction.getPathLoadedFrom().equals(metadata.getPath())) {
+      throw new KsqlException("Can't add function " + ksqlFunction
+          + "as a function with the same name has been loaded from a different path "
```

Comment:
wouldn't it be more accurate to say `... has been loaded from a different jar`, especially since the full path of the jar is going to be included in the error message?